Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -1988,8 +1988,7 @@ class WebCore::TransformOperations {
     std::optional<WebCore::SourceBrush::Brush> brush();
 }
 
-#if ENABLE(GPU_PROCESS)
-
+#if ENABLE(GPU_PROCESS) && ENABLE(WEBGL)
```

Comment:
This could probably just be ENABLE(WEBGL).  The declaration has no ENABLE(GPU_PROCESS)

Reply:
I don’t think serialization is needed unless ENABLE(GPU_PROCESS)